.prettierrc.js

```diff
@@ -3,5 +3,6 @@ module.exports = {
   trailingComma: 'all',
   singleQuote: true,
   printWidth: 100,
-  tabWidth: 2
+  tabWidth: 2,
+  endOfLine: 'auto'
```

Comment:
What does this do?

Reply:
Its to ignore the type of return between operation systems.  There are more information here.
https://stackoverflow.com/questions/53516594/why-do-i-keep-getting-delete-cr-prettier-prettier

Reply:
Interesting... I didn't know prettier was capable of this. Looks good, let's keep it.